crates/holochain_p2p/src/spawn/actor.rs

```diff
@@ -2139,6 +2145,37 @@ impl actor::HcP2p for HolochainP2pActor {
                 .collect())
         })
     }
+
+    fn conductor_db_getter(&self) -> GetDbConductor {
+        self.blocks_db_getter.clone()
+    }
+
+    fn block(
```

Comment:
I was torn between keeping this generic over `BlockTarget` and making it specifically a `CellBlockTarget`. With the generic block target new variants can be added without touching this API, the specific cell block function could be complemented by new functions if there were ever more ways of blocking in holochain_p2p.

Reply:
I prefer the `BlockTarget` as it is